recipes/bytewax-feedstock/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "bytewax" %}
+{% set version = "0.16.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/bytewax/bytewax/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 993fe70f3a206adc52aae1099c00e14a92ec55387dda4b0cda2b6f6fbf0ed35b
+  patches:
+    # this patch disable the vendored ssl/gssapi vendored libraries
+    # falling back to use the conda installed ones
+    - ssl.patch
+
+build:
+  skip: True  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - curl
+    - {{ compiler('cxx') }}
+    - make
+    - pkg-config
+    - python
```

Comment:
In theory you should only need `{{ compiler('cxx') }}` in build. Sometimes, when cross-compiling, you may need Python and/or others. Can you try to remove these and/or move to `host` to make sure you need them here?

Reply:
done